### PR TITLE
Add docs for the events plugin

### PR DESCRIPTION
**Pull request by alessio**
The PR's title should be self-explanatory.
Comments and fixes would be much appreciated!

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Docker powered mini-Heroku. The smallest PaaS implementation you've ever seen.
 - [Nginx Configuration](http://progrium.viewdocs.io/dokku/nginx)
 - [Running Remote commands](http://progrium.viewdocs.io/dokku/remote-commands)
 - [Container Options](http://progrium.viewdocs.io/dokku/docker-options)
+- [Server logs] (http://progrium.viewdocs.io/dokku/logging)
```
Comment (michaelshobbs) [inline docs/index.md]: I think "server logs" might be ambiguous here. Perhaps we can say "dokku logs"?

**Comment by alessio** [comment]
Done @michaelshobbs

File: docs/index.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ Docker powered mini-Heroku. The smallest PaaS implementation you've ever seen.
 - [Nginx Configuration](http://progrium.viewdocs.io/dokku/nginx)
 - [Running Remote commands](http://progrium.viewdocs.io/dokku/remote-commands)
 - [Container Options](http://progrium.viewdocs.io/dokku/docker-options)
+- [Dokku events logs] (http://progrium.viewdocs.io/dokku/logging)
```
Comment (josegonzalez) [inline docs/index.md]: Uppercase the `e` in `events`.

File: docs/logging.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+```
+events [-t]                                     Show the last events (-t follows)
+events:list                                     List logged events
+events:on                                       Enable events logger
+events:off                                      Disable events logger
+```
+
+## System logs
+
+The _events_ plugin writes log entries to ``/var/log/syslog`` as well as a Dokku-specific logs sub-directory ``/var/log/dokku/events.log``.
```
Comment (josegonzalez) [inline docs/logging.md]: Maybe add an example of output for each command as well as the contents of the log file itself?

**Pull request merged**